### PR TITLE
fix(FileInput): add text ellipsis for long filenames

### PR DESCRIPTION
**Pull request by tenphi**

### CODE REVIEW

File: src/components/forms/FileInput/FileInput.stories.tsx [hunk 1]
```diff
@@ -13,7 +13,11 @@ export default {
 };
 
 const Template = ({ icon, ...props }) => (
-  <FileInput {...props} onChange={(query) => console.log('change', query)} />
+  <FileInput
+    {...props}
+    inputStyles={{ width: '300px' }}
```
Comment (MrFlashAccount) [inline src/components/forms/FileInput/FileInput.stories.tsx:18]: Put this in args please.

File: src/components/forms/FileInput/FileInput.stories.tsx [hunk 2]
```diff
@@ -13,7 +13,11 @@ export default {
...
+    onChange={(query) => console.log('change', query)}
```
Comment (MrFlashAccount) [inline src/components/forms/FileInput/FileInput.stories.tsx:19]: Btw, guess we should fix it.

File: src/components/forms/FileInput/FileInput.tsx [hunk 3]
```diff
@@ -77,7 +82,7 @@ const FileInputElement = tasty(Action, {
       bottom: 0,
       left: 0,
       radius: '@content-radius',
-      // opacity: 0,
```
Comment (MrFlashAccount) [inline src/components/forms/FileInput/FileInput.tsx:80]: Commenting out?

**Review by MrFlashAccount — CHANGES_REQUESTED** [review]

**Comment by MrFlashAccount** [comment]
Please add a story that tests the overflow behavior.

It's possible with testing library https://testing-library.com/docs/user-event/utility

File: src/components/forms/FileInput/FileInput.stories.tsx [hunk 4]
```diff
@@ -12,9 +17,43 @@ export default {
...
+    qa="FileInput"
+    {...props}
+    onChange={(value) => console.log('onChange', value)}
+  />
 );
 
 export const Default = Template.bind({});
 Default.args = {};
+
+export const LongFilePlaceholderOverflow: typeof Template = Template.bind({});
```
Comment (MrFlashAccount) [inline src/components/forms/FileInput/FileInput.stories.tsx:31]: Тип можно не писать, TS умеет выводить самостоятельно :)

**Review by MrFlashAccount — APPROVED** [review]

File: src/components/forms/FileInput/FileInput.stories.tsx (hunk 4 above)
Comment (tenphi) [reply src/components/forms/FileInput/FileInput.stories.tsx:31]: I had a weird typing issue and tried things :) I will remove it.
